### DISCUSSION TITLE
[Feature Request] Add moto tag support

### DISCUSSION THREAD

**Discussion by Wii505** [seemoo-lab/AirGuard#193] [General]
Motorola has made their own [trackers](https://www.motorola.com/us/en/p/motoaccessories/all-moto-tags/pg38c06069?pn=PG38C06069)

**Comment by imbissbudenaesthetik** [comment]
Hi, this feature has been implemented. AirGuard can see Google Find My Device Trackers, including the Moto Tag; there is even dedicated support to detect a Moto Tag if the user connects to it using the detailed scan.
